tool/tctl/common/resource_command.go

```diff
@@ -782,6 +794,12 @@ func (rc *ResourceCommand) Delete(ctx context.Context, client auth.ClientI) (err
 			return trace.Wrap(err)
 		}
 		fmt.Printf("%s '%s/%s' has been deleted\n", types.KindCertAuthority, rc.ref.SubKind, rc.ref.Name)
+	case types.KindInstaller:
+		err := client.DeleteInstaller(ctx)
+		if err != nil {
+			return trace.Wrap(err)
+		}
+		fmt.Printf("%s has been reset to a default value\n", types.KindInstaller)
```

Comment:
Where is the logic that actually returns the default installer value if it's not set? I either didn't see it or missed it. I only saw that logic that sets installer if it's not set on the init which is not the correct place for it because resource can still be deleted and won't be recreated until after restart.